src/config/dynamic-theme-fixes.config

```diff
@@ -620,6 +620,15 @@ INVERT
 
 ================================
 
+bbb*
```

Comment:
What's an example URL?

Reply:
Log in to a demo at https://demo.bigbluebutton.org.